src/components/MainView.js

```diff
@@ -19,6 +20,7 @@ function MainView({ children }) {
         `}
       >
         <Header />
+        <Banner />
```

Comment:
Per the previous comment, I think this should live inside of `Proposals`. That'll also keep it inside the scrollable area.